Command/Async/ProcessOrderCommand.php

```diff
@@ -0,0 +1,132 @@
+
+        parent::__construct($name);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function configure()
+    {
+        $this->setName('amazon:payment:order:process')
+            ->addArgument(
+                'increment_id',
+                InputArgument::IS_ARRAY | InputArgument::REQUIRED,
+                'Order increment id(s) to process (separate multiple with spaces)'
+            );
+        parent::configure();
+    }
+
+    /**
+     * Execute asynchronous processing of pending orders
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->state->setAreaCode(\Magento\Framework\App\Area::AREA_ADMINHTML);
+        $incrementIds = (array) $input->getArgument('increment_id');
+
+        $connection = $this->resource->getConnection();
+
+        foreach ($incrementIds as $incrementId) {
+            $output->writeln("<info>Processing order: {$incrementId}</info>");
+
+            $txnSelect = $connection->select()
+                ->from($connection->getTableName('sales_payment_transaction'), ['txn_id'])
+                ->joinLeft(
+                    ['so' => $connection->getTableName('sales_order')],
+                    'so.entity_id = sales_payment_transaction.order_id',
+                    []
+                )
+                ->where('so.increment_id = ?', $incrementId);
+
+            $txnIds = $connection->fetchCol($txnSelect);
+
+            if (empty($txnIds)) {
+                $output->writeln("<comment>No payment transactions found for order {$incrementId}</comment>");
+                continue;
+            }
+
+            foreach ($txnIds as $txnId) {
+                $output->writeln("Found txn_id: {$txnId}");
+
+                // 2. Get matching async records
+                $collection = $this->asyncCollectionFactory->create();
+                $collection->addFieldToFilter(AsyncInterface::PENDING_ID, $txnId);
+                $collection->addFieldToFilter(AsyncInterface::IS_PENDING, ['eq' => 1]);
+
+                if ($collection->getSize() === 0) {
+                    $output->writeln("<comment>No async records found for txn_id {$txnId}</comment>");
+                    continue;
+                }
+
+                foreach ($collection as $item) {
+                    /** @var \Amazon\Pay\Model\Async $item */
+                    $output->writeln("<info>Processing async entity #{$item->getId()} for txn_id {$txnId}</info>");
+                    try {
+                        $this->asyncUpdater->processPending($item);
+                    } catch (\Exception $e) {
+                        $output->writeln("<error>Error processing {$txnId}: {$e->getMessage()}</error>");
+                    }
+                }
+            }
+        }
+        $output->writeln("<info>Finish order processing</info>");
```

Comment:
Let's update `Finish` to `Finished`.

Reply:
Updated here https://github.com/BearGroup/amazon-payments-magento-2-plugin/pull/125/commits/27629c05688df25929b91d8fe540eac188c966c0